### PR TITLE
ignore_suffix as skywalking CDS target

### PR DESCRIPTION
**Pull request by Shikugawa**
`ignore_suffix` as skywalking CDS target and removed `instance_name` from dynamic configurable properties.

### CODE REVIEW

**Comment by Shikugawa** [comment]
I will release v0.2.0 after this merged.

File: README.md [hunk 1]
```diff
@@ -136,19 +136,25 @@ tracer->report(std::move(tracing_context));
...
 config.set_cds_request_interval(5); // CDS request interval should be 5sec
 ```
 
-Currently, Configurable values dynamically are like below.
+If you are using Consul KVS as backend, we could put configuration value through HTTP request.
 
-| Config Key | Value Description |
-|:----:|:----:|
-|instance_name| Instance name of this agent |
+```config.yaml
```
Comment (wu-sheng) [inline README.md:149]: I think `config.yaml` doesn't show up on the GitHub preview page, https://github.com/SkyAPM/cpp2sky/blob/0d8bb7949686c8c6c6c3baba188a7c6b18be2a5e/README.md

File: docs/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@
 | address | OAP address. | string | string |
 | token | OAP token. | string | string |
 | delayed_buffer_size | The size of buffer it stores pending messages. | uint32 | uint32 |
-| ignore_operation_name_suffix | If the operation name of the first span is included in this set, this segment should be ignored. | (slice of) string | (slice of) string |
+| ignore_operation_name_suffix | If the operation name of the first span is included in this set, this segment should be ignored. This value can be changed with SkyWalking CDS. | (slice of) string | (slice of) string |
```
Comment (wu-sheng) [inline docs/README.md:14]: Adding a new column in the table should be better?

**Review by wu-sheng — APPROVED** [review]

File: docs/README.md (hunk 2 above)
Comment (Shikugawa) [reply docs/README.md]: I think it is a pretty terrible protodoc hack... But it is "ok", I thought. https://github.com/SkyAPM/cpp2sky/pull/66/files#diff-2df70a32cd330081cbde9cddb5cda95b1355c9f5dd5155270f7d955414dfc14e

Comment (wu-sheng) [reply docs/README.md]: It is good to me. I am new to this :)

**Pull request merged**